Gemfile

```diff
@@ -80,6 +80,11 @@ gem "clipboard-rails"
 # One-time passwords for 2fa
 gem "rotp", "~> 3.3"
 
+# WHOIS lookup for unverified publishers
+gem "whois", "~> 4.0"
```

Comment:
could you update `Gemfile.lock` as a result of the new gems?

Reply:
also, you could probably do `require: false` for this (then require it manually)